taxi/models.py

```diff
@@ -11,18 +12,30 @@ class Meta:
         ordering = ["name"]
 
     def __str__(self):
-        return f"{self.name} {self.country}"
+        return f"{self.name} ({self.country})"
 
 
 class Driver(AbstractUser):
-    license_number = models.CharField(max_length=255, unique=True)
+    license_number = models.CharField(
+        max_length=8,
+        unique=True,
+        help_text="Exactly 3 uppercase"
+                  " letters followed by 5 digits (e.g. ABC12345).",
+        validators=[
+            RegexValidator(
+                regex=r"^[A-Z]{3}\d{5}$",
```

Comment:
you have that regex in constant at another file, maybe better to make a file with constants or import that variable here?